mbuild/tests/test_compound.py

```diff
@@ -15,6 +15,33 @@ class TestCompound(BaseTest):
     def test_load_and_create(self):
         mb.load(get_fn('methyl.pdb'))
 
+    def test_load_conversion(self,ethane,h2o):
+        compound = mb.Compound([ethane,h2o])
+        parm = compound.to_parmed()
+        traj = compound.to_trajectory()
+
+        for topo in [compound,parm,traj]:
+            topo_converted = mb.load(topo)
+            assert isinstance(topo_converted, mb.Compound)
+            assert topo_converted.n_particles == 11
+            assert len([at for at in topo_converted.particles() if at.name == 'C']) == 2
+            assert len([at for at in topo_converted.particles() if at.name == 'H']) == 8
+            assert len([at for at in topo_converted.particles() if at.name == 'O']) == 1
+
+        for topo in [parm,traj]:
+            new_topo = mb.load(compound)
+            new_topo.xyz = np.random.random(topo_converted.xyz.shape)
+            new_topo = mb.load(topo, compound=new_topo, coords_only=True)
+            assert np.allclose(mb.load(topo).xyz, new_topo.xyz)
+
+        # Extra test (test files from Parmed/test/files/test.mol2)
+        test = pmd.load_file(get_fn('test.mol2'),structure=True)
```

Comment:
Is there a reason we need such a large file? Is there something being tested here that required 10,000 atoms?

Reply:
Yeah, I have no issue with that, last time I just went a picked a random topology file from the internet and it just turned out to be a 10,000 atoms file. I can try to use a different (and smaller) file for testing (probably one of those that we have over foyer).